gix-discover/src/lib.rs

```diff
@@ -39,6 +39,8 @@ pub mod is_git {
         Metadata { source: std::io::Error, path: PathBuf },
         #[error("The repository's config file doesn't exist or didn't have a 'bare' configuration or contained core.worktree without value")]
         Inconclusive,
+        #[error("Could not obtain current directory when conforming repository path")]
```

Comment:
Is this, and the related `conformed_git_dir` variable, about something conforming to a standard or expectation? Or are they typos, where "confirming" and `confirmed_git_dir` should be written instead?

If "conform" is the intended word, then I suggest rephrasing this message for clarity and probably also renaming the `conformed_git_dir` variable to `conforming_git_dir`. If "confirm" is intended--i.e. if the intended meaning is that we are verifying that something is a git dir--then I recommend changing this to "confirming" and renaming the variable to `confirmed_git_dir`.

I'm not sure which meaning is intended, so I haven't proposed a specific edit at this time. Also, I couldn't propose it in a review comment easily, because at least as applied to the variable name, this is a preexisting ambiguity, and some occurrences of the variable name are in places this PR does not modify. If it is known that one of these two changes is desired, though, then I'd be pleased to open a PR for them ~~after~~ now that this PR is merged.

Reply:
Thanks for the hint - the affected variable name is definitely leaking here and I improved the message to be much more specific [in this PR](https://github.com/Gitoxide-Labs/gitoxide/pull/1618).